test/unit/com/unifina/filters/UnifinaCoreAPIFiltersSpec.groovy

```diff
@@ -57,4 +61,32 @@ class UnifinaCoreAPIFiltersSpec extends Specification {
 		then:
 		response.status == 200
 	}
+
+	void "authentication passes when session token provided"() {
+		given:
+		defineBeans {
+			sessionService(MethodInvokingFactoryBean) {
```

Comment:
Not 100% sure what's happening here, but if you want to replace beans with whatever mocks or stubs, you could make the test class extend `BeanMockingSpecification` instead of `Specification` and use some of the utils in there